backend/test_db_connection.py

```
@@ -0,0 +1,76 @@
+"""
+import sys
+import os
+sys.path.insert(0, os.path.join(os.path.dirname(__file__)))
+
+from app.api.deps import get_db
+from sqlalchemy import text
+
+def test_database():
+    """测试数据库连接和表状态"""
+    print("🔍 测试数据库连接...")
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)